numpy/core/arrayprint.py

```diff
@@ -622,8 +622,13 @@ def array2string(a, max_line_width=None, precision=None,
     return _array2string(a, options, separator, prefix)
 
 
-def _extendLine(s, line, word, line_width, next_line_prefix):
-    if len(line) + len(word) > line_width:
+def _extendLine(s, line, word, line_width, next_line_prefix, legacy):
+    needs_wrap = len(line) + len(word) > line_width
+    if legacy != '1.13' and len(line) == len(next_line_prefix):
```

Comment:
I suppose strictly this should be `<=`, but in practice that never happens

Reply:
Sure you don't want to change it before this goes in?

Reply:
Guess I will. Updated